org.purei.Play.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: org.purei.Play
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: Play
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=home:ro
+  - --filesystem=/mnt:ro
+  - --filesystem=/media:ro
+  - --filesystem=/run/media:ro
+  - --device=all
+  - --socket=pulseaudio
+modules:
+  - "shared-modules/glu/glu-9.json"
+  - name: evdev
+    config-opts: 
+      - --disable-static
+    cleanup: 
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /lib/*.la
+      - /share/man
+    sources:
+      - type: archive
+        url: https://www.freedesktop.org/software/libevdev/libevdev-1.12.1.tar.xz
+        sha256: 1dbba41bc516d3ca7abc0da5b862efe3ea8a7018fa6e9b97ce9d39401b22426c
+        x-checker-data:
+          type: anitya
+          project-id: 20540
+          stable-only: true
+          url-template: https://www.freedesktop.org/software/libevdev/libevdev-$version.tar.xz
+  - name: Play!
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    post-install:
+      - |
+        GIT_VERSION=$(git describe --always);
+        GIT_DATE=$(git show -s --format=%cs);
+        sed -i 's|<releases>|<releases>\n    <release date="'${GIT_DATE}'" version="'${GIT_VERSION}'"/>|' ${FLATPAK_DEST}/share/metainfo/org.purei.Play.metainfo.xml;
```

Comment:
The existing version can be replaced, it is essentially a place holder.
Also there is no version, I don't know what `$GIT_VERSION` will be.
Maybe call it `nightly`.

Reply:
GIT_VERSION is meant to be `VERSION-HASH` but its not correct in the build, probably because build system doesnt pull all the tags, latest comment should fix that.

Reply:
A commit hash is a developer information, not really one for the end user.
`nightly`, `latest` or similar will look better, and will actually mean something for them.
Note that the commit is already present in the Flatpak metadata (`flatpak info`).

Just in a single line is at the end needed in `post-install`:
`sed -i "s/date=.* v/date=\"$(git show -s --format=%cs)\" version=\"nightly\"/" ${FLATPAK_DEST}/share/metainfo/org.purei.Play.metainfo.xml`

Reply:
this is the versioning we show within the app, and the one we expect during reports

Reply:
Ok. Why still keeping the existing release instead of replacing it, which will likely not be updated (unless you handle that).

Reply:
its just easier to remove that line upstream than worry about it from this side.
its also easier to fix the current issue (this way) on this side than upstream (for now), I think this entire post-install can be moved upstream into the cmake script itself.

Reply:
Maybe but then it won't pass the `flatpak run org.freedesktop.appstream-glib validate org.purei.Play.metainfo.xml` linter.
It can be considered as a placeholder.

Note also that versions **must** be sorted. Commit hashes not being version, CI errors on the Flathub side may happen in the future if older releases are kept.,

Reply:
IMHO upstream should be the proper version when doing a release. Remember, flathub is to distribute releases.

Reply:
even upstream version is set per commit/build, so it works out the same, Im merely saying that managing this upstream makes more sense, since it simplified the script on this side, but it'd also technically needed for appimage so no need for duplicate work, though this is something I'd do down the line (busy for the next few weeks)

Reply:
Anyway this shouldn't block the merge?
As @jpd002 whish, this Flatpak package should track master.
@Zer0xFF and @jpd002 what do you think of version like `0.50-10a3d86` (version-shorthash)? That's what [xemu](https://github.com/mborgerson/xemu/releases) does.